### PR TITLE
feat: Create UI table show list customer

### PR DESCRIPTION
**Pull request by Lanng21**
# Description

- Create UI table show list customer 

image

## Detailed PR Verification Procedure(s)

- Go to `/components/common` to check the UI for table show list customer

## Ticket

[GS-34](https://tito-training.atlassian.net/browse/GS-34)

## Application Checklist

- [x] **JIRA Linked**: Author has followed template and has a title in the following format:
  > GS-34: Create UI table show list customer
- [ ] **No Breaking Changes**: Author has confirmed all API contract changes are non-breaking.
- [ ] **Changes Unit Tested**: Author has unit tested new code to prove it works.
- [ ] **Application Passes All Tests**: Author ensures this pull-request passes `npm test`.
- [x] **Author Self Reviewed**: Author of pull-request has self-reviewed this code.
- [x] **Author is Assignee**: Author has assigned self as assignee of pull request.

## Security Checklist

- [ ] **Code Review for Security:** Developer has self-reviewed this code, based on OWASP Top-10 vulnerabilities and general environment security attributes, to avoid dversely impact the security of the system/environment/application, to the best of their abilities.
- [ ] **Backout Procedures:** Developer has considered how the changes presented in this pull request can be replaced with a prior version of these changes, fixed going forward, and/or implemented to be backwards compatible.

### CODE REVIEW

File: src/views/Constants/customer.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
+//type
+import { Idata } from '@/views/types/table'
+
+export const data: Idata[] = [
```
Comment (baonguyen26) [inline src/views/Constants/customer.ts]: Should move it to `dataMock` folder

File: src/views/components/index.tsx [hunk 2]
```diff
@@ -1,7 +1,58 @@
 import React from 'react'
+import Table from './common/Table'
```
Comment (baonguyen26) [inline src/views/components/index.tsx]: Why is it a file in the components folder?

File: src/views/dataMocks/customer.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
+//type
```
Comment (baonguyen26) [inline src/views/dataMocks/customer.ts]: This file should be outside of the views folder

File: src/views/types/table/index.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+export interface Idata {
```
Comment (baonguyen26) [inline src/views/types/table/index.ts]: This file should be outside of the views folder

File: src/constants/Customer.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+const Table = {
+  stt: 'STT',
+  name: 'Full name',
+  phoneNumber: 'Phone Number',
+  staff: 'Staff in charge',
+  demand: 'Demand',
+  createdAt: 'Date Created',
```
Comment (haunguyen15-tt) [inline src/constants/Customer.ts]: Need translation, right ?

**Review by baonguyen26 — APPROVED** [review]

**Pull request merged**